packages/gestalt/src/Text.js

```diff
@@ -52,6 +53,10 @@ type Props = {|
    * Indicates how the text content should handle overflowing its container. See the [overflow and truncation variant](https://gestalt.pinterest.systems/web/text#Overflow-and-truncation) for more details.
    */
   overflow?: Overflow,
+  /**
+   * Ref that is forwarded to the underlying element. See the [ref variant](https://gestalt.pinterest.systems/web/text#Refs) for more details.
+   */
+  ref?: HTMLDivElement | HTMLSpanElement,
```

Comment:
Huh…usually we have to disable `react/no-unused-prop-types` as Flow expects `ref` to be destructured with the rest of the props. I wonder what the difference is here?

Reply:
I noticed that too. There are other examples where it doesn't complain.